<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMissingEpisodes</class>
 <widget class="QWidget" name="CMissingEpisodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Missing Episodes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="dataSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2" colspan="2">
        <widget class="QDateEdit" name="minPremiereDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="8">
        <widget class="QTreeView" name="servers">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QCheckBox" name="searchByShowName">
         <property name="text">
          <string>Show Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QDateEdit" name="maxPremiereDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="searchByDate">
         <property name="text">
          <string>Premiere Date Range:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="8">
        <widget class="QLabel" name="mediaSummaryLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Media Summary:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="8">
        <widget class="QLabel" name="serversLabel">
         <property name="text">
          <string>Servers:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QComboBox" name="shows">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="placeholderText">
          <string>Shows Must Be Loaded</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
